api/models/dataSource.js

```diff
@@ -0,0 +1,12 @@
+const { DataSource } = require("typeorm");
+
+const dataSource = new DataSource({
+  type: "mysql",
```

Comment:
```suggestion
  type: process.env.DB_CONNECTION,
```
`"mysql"`이라고 하드코딩 하지않고, `.env`속의 환경변수를 활용해주세요~